Google.Api.Generator.Tests/ProtoTests/UnitTests/UnitTests.proto

```diff
@@ -0,0 +1,84 @@
+﻿syntax = "proto3";
+
+package testing.unittests;
+
+option csharp_namespace = "Testing.UnitTests";
+
+import "google/api/annotations.proto";
+
+service UnitTests {
+  option (google.api.default_host) = "unittests.example.com";
+
+  rpc MethodValues(ValuesRequest) returns(Response);
+}
+
+option (google.api.resource_definition) = {
+  name: "AResource",
+  path: "items/{item_id}/parts/{part_id}"
+};
+
+option (google.api.resource_definition) = {
+  name: "WildcardResource",
+  path: "*"
+};
+
+message AnotherMessage {
+}
+
+enum Enum {
+  DEFAULT = 0; // First value must be 0 and is the default
+}
+
+message ValuesRequest {
```

Comment:
Note that this doesn't include a oneof with two fields. I haven't looked terribly closely at the generation code, but my guess is that if it *did* include a oneof, we'd end up generating a test with an odd proto.

Reply:
Right, yes, thanks.
I'll put a TODO in now, and add proper one-of support in a future PR.
The current (GAPIC monolith) test generation also doesn't handle one-ofs properly.